projects/ngx-sub-form/src/lib/ngx-error-default-messages.ts

```diff
@@ -0,0 +1,17 @@
+import { IFormatters } from "./ngx-error.pipe";
+
+function format(msg: string, displayName?: string) {
+    return displayName ? `${displayName || ''} ${(msg || '').toLowerCase()}` : msg;
+}
+
+export const errorDefaultMessages: IFormatters = {
```

Comment:
these can be overridden by passing an argument to the `subFormErrorProvider` helper in utils